### PR TITLE
Add 'nexus-fixer'

### PR DESCRIPTION
**Pull request by hygt**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -35,10 +43,12 @@ pipeline {
         }
         stage("Build & Publish Artifacts") {
             when {
-                expression { !isPR }
+                expression { isPR }
```
Comment (bogdanromanx) [inline Jenkinsfile]: is this change intentional?

File: permissions-fixer/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+[package]
+name = "nexus-fixer"
+version = "1.0.0"
+authors = ["Henry Genet <henry.genet@epfl.ch>"]
+edition = "2018"
```
Comment (bogdanromanx) [inline permissions-fixer/Cargo.toml:5]: 2018?

File: permissions-fixer/src/errors.rs [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+use Failure::*;
+
+#[derive(Eq, PartialEq)]
+pub enum Failure {
+    PathCannotBeEmpty,
+    PathMustBeAbsolute,
+    PathMustBeCanonical,
+    PathMustStartWithPrefix,
+    PathCannotHaveNul,
```
Comment (bogdanromanx) [inline permissions-fixer/src/errors.rs]: Null?

**Review by bogdanromanx — CHANGES_REQUESTED** [review]

File: Jenkinsfile (hunk 1 above)
Comment (hygt) [reply Jenkinsfile]: Yes to test the Jenkinsfile ;) I reverted.

File: permissions-fixer/Cargo.toml (hunk 2 above)
Comment (hygt) [reply permissions-fixer/Cargo.toml:5]: It's the Toml manifest format, 2018 is the latest version

File: permissions-fixer/src/errors.rs (hunk 3 above)
Comment (hygt) [reply permissions-fixer/src/errors.rs]: The `\0` character is called NUL in ASCII. Anyway this shouldn't happen.

**Review by bogdanromanx — DISMISSED** [review]

**Review by umbreak — DISMISSED** [review]

**Review by umbreak — APPROVED** [review]
